x/ccv/provider/keeper/relay.go

```diff
@@ -343,10 +343,31 @@ func (k Keeper) OnRecvSlashPacket(
 		return ccv.V1Result, nil
 	}
 
-	// Check that chain is launched and the validator belongs to the consumer chain valset
-	if k.GetConsumerPhase(ctx, consumerId) == providertypes.CONSUMER_PHASE_LAUNCHED && !k.IsConsumerValidator(ctx, consumerId, providerConsAddr) {
-		k.Logger(ctx).Error("cannot jail validator %s that does not belong to consumer %s valset",
-			providerConsAddr.String(), consumerId)
+	// check that the chain is launched
+	if k.GetConsumerPhase(ctx, consumerId) != providertypes.CONSUMER_PHASE_LAUNCHED {
+		k.Logger(ctx).Error("cannot jail validator on a chain that is not currently launched",
+			"consumerId", consumerId,
+			"provider cons addr", providerConsAddr.String(),
+		)
+
+		// Naturally, we do not return a slash ack here because the fact that we are here means:
```

Comment:
I'm not sure I understand this point. Are you referring to `AppendSlashAck`?

Reply:
Yes, I'm referring to `AppendSlashAck`. It's also related to your comment [above](https://github.com/cosmos/interchain-security/pull/2346#discussion_r1799093649).

What I practically want to say, is that this chain has been stopped and is to be removed. Any future slash packets won't be handled because the chain is not launched anymore.
Yes, a stopped consumer chain itself might still be able to send slash packets to the provider, but technically we do not really expect those slash packets and we should not send a slash ack back to the consumer chain.

Reply:
That true. However, I would still append the slash acks. These will not be send anyway as the chain is stopped (no VSCPackets are sent for stopped chains). However, if we later change the logic to move a consumer from stopped back to launched, then this might complicate things. The slash acks are deleted when the chain is removed, right?

Reply:
Not that important though. Either way would work now.

Reply:
Good point. Let me readd the slash acks.
> The slash acks are deleted when the chain is removed, right?
[Yes.](https://github.com/cosmos/interchain-security/blob/v6.1.0/x/ccv/provider/keeper/consumer_lifecycle.go#L481)